digdag-docs/src/operators/require.md

```diff
@@ -74,7 +74,7 @@
 
   *rerun_on* control require> really kicks or not if the attempt for the dependent workflow already exists. 
   * *none* ... Not kick the workflow if the attempt already exists.
-  * *failed* ... Kick the workflow if the attempt exists and its result is not success.
+  * *failed* ... Kick the workflow if the attempt not exists or its result was not success.
```

Comment:
I don't think _"attempt not exists"_ is proper because this option assumes existence of the attempt as the following description:
> *rerun_on* control require> really kicks or not if the attempt for the dependent workflow already exists.

Reply:
In the actual behavior, `rerun_on: failed` will generate a new attempt even if the attempt_id at the corresponding session_time has not been issued.  so, how about this description?
```
*failed* ... Kick the workflow if the attempt may exists and its result is not success. 
```